### PR TITLE
Translation: locales/fr.json

### PR DESCRIPTION
**Pull request by sven-hash**

### CODE REVIEW

File: src/lib/locales/fr.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "title" : "Nostr",
+  "nav" : {
+    "resources" : "Ressources",
+    "whatIs" : "Qu'est-ce que Nostr ?",
```
Comment (erskingardner) [inline src/lib/locales/fr.json:5]: Can we drop the space between the text and the `?`?

**Review by erskingardner — CHANGES_REQUESTED** [review]

**Comment by erskingardner** [comment]
Thanks @sven-hash – check out the readme on translations, there are a bunch of markdown pages that will need to be created before we can put french live. But, in the meantime, we can merge this .json file.

**Comment by sven-hash** [comment]
@erskingardner thanks for your quick answer, I've made another PR for this and the guide will come very soon

File: src/lib/locales/fr.json (hunk 1 above)
Comment (erskingardner) [reply src/lib/locales/fr.json:5]: Ok – let's keep it then! 👍

**Review by erskingardner — APPROVED** [review]

**Pull request merged**